src/containers/JudgingPanel.js

```diff
@@ -108,24 +108,30 @@ const calculateResiduals = project => {
 
 const getProjectData = async () => {
   const projectDocs = await projectsRef.get()
-  return projectDocs.docs.map(projectDoc => projectDoc.data())
+  return projectDocs.docs.map(projectDoc => {
+    return { ...projectDoc.data(), id: projectDoc.id }
```

Comment:
why is this explicit spread needed?

Reply:
data doesnt come with id and id doesnt come with data